styles/style.css

```diff
@@ -36,7 +28,19 @@ button {
 
 /* Buttons */
 /* INSERT BUTTON STYLES HERE */
+button.volunteer-button {
```

Comment:
This styling is great, but you need to add the `volunteer-button` class to the `button` element in your HTML.